[2025-10-20T04:45:08.218+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-20T04:45:08.282+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flood_prediction_pipeline.fetch_weather_data manual__2025-10-20T04:45:05.684164+00:00 [queued]>
[2025-10-20T04:45:08.302+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flood_prediction_pipeline.fetch_weather_data manual__2025-10-20T04:45:05.684164+00:00 [queued]>
[2025-10-20T04:45:08.304+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-20T04:45:08.331+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_weather_data> on 2025-10-20 04:45:05.684164+00:00
[2025-10-20T04:45:08.351+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=296) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-20T04:45:08.346+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'flood_prediction_pipeline', 'fetch_weather_data', 'manual__2025-10-20T04:45:05.684164+00:00', '--job-id', '558', '--raw', '--subdir', 'DAGS_FOLDER/flood_prediction_dag.py', '--cfg-path', '/tmp/tmpey568fyu']
[2025-10-20T04:45:08.352+0000] {standard_task_runner.py:63} INFO - Started process 311 to run task
[2025-10-20T04:45:08.353+0000] {standard_task_runner.py:91} INFO - Job 558: Subtask fetch_weather_data
[2025-10-20T04:45:08.436+0000] {task_command.py:426} INFO - Running <TaskInstance: flood_prediction_pipeline.fetch_weather_data manual__2025-10-20T04:45:05.684164+00:00 [running]> on host 7a672faa1ea7
[2025-10-20T04:45:08.584+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flood_prediction_pipeline' AIRFLOW_CTX_TASK_ID='fetch_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-10-20T04:45:05.684164+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-20T04:45:05.684164+00:00'
[2025-10-20T04:45:08.586+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-20T04:45:08.638+0000] {logging_mixin.py:188} INFO - âœ… Weather data fetched: {'temp': 21.34, 'humidity': 88.93, 'pressure': 1020.72, 'wind_speed': 0.45, 'rain_1h': 5.58, 'rain_3h': 40.34, 'rain_6h': 138.66, 'rain_24h': 166.61, 'rain_intensity': 8.61, 'temp_delta': -3.5, 'humidity_delta': 5.010000000000005, 'hour': 4, 'minute': 45, 'second': 8, 'day': 20, 'month': 10, 'year': 2025}
[2025-10-20T04:45:08.639+0000] {python.py:237} INFO - Done. Returned value was: {'temp': 21.34, 'humidity': 88.93, 'pressure': 1020.72, 'wind_speed': 0.45, 'rain_1h': 5.58, 'rain_3h': 40.34, 'rain_6h': 138.66, 'rain_24h': 166.61, 'rain_intensity': 8.61, 'temp_delta': -3.5, 'humidity_delta': 5.010000000000005, 'hour': 4, 'minute': 45, 'second': 8, 'day': 20, 'month': 10, 'year': 2025}
[2025-10-20T04:45:08.640+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-20T04:45:08.681+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=flood_prediction_pipeline, task_id=fetch_weather_data, run_id=manual__2025-10-20T04:45:05.684164+00:00, execution_date=20251020T044505, start_date=20251020T044508, end_date=20251020T044508
[2025-10-20T04:45:08.730+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-20T04:45:08.754+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-20T04:45:08.761+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
